database/submit_to_database.py

```diff
@@ -479,6 +481,11 @@ def flux_cal_and_submit(time_detection, time_obs, metadata, bestprof_data,
         logger.error("Please us either (--obsid and --pulsar) or --bestprof")
         quit()
 
+    if args.pulsar:
+        #Overrule what's in the bestprof if pulsar name is supplied
+        pulsar = args.pulsar
+    print("PULSAR NAME: {}".format(pulsar))
```

Comment:
Use logger.info

Reply:
Used debug, since this was why i put this test here in the first place